blockbase/functions.php

```diff
@@ -187,3 +134,99 @@ function blockbase_core_navigation_render_fallback() {
 if ( file_exists( get_stylesheet_directory() . '/inc/block-patterns.php' ) ) {
 	require_once get_stylesheet_directory() . '/inc/block-patterns.php';
 }
+
+
+// ----------	Custom Fonts ---------- //
+const BLOCKBASE_GOOGLE_FONTS_LIST = array(
+	'Arvo',
+	'Bodoni Moda',
+	'Cabin',
+	'Chivo',
+	'Courier Prime',
+	'DM Sans',
+	'Domine',
+	'EB Garamond',
+	'Fira Sans',
+	'IBM Plex Sans',
+	'IBM Plex Mono',
+	'Inter',
+	'Josefin Sans',
+	'Jost',
+	'Libre Baskerville',
+	'Libre Franklin',
+	'Literata',
+	'Lora',
+	'Merriweather',
+	'Montserrat',
+	'Newsreader',
+	'Nunito',
+	'Open Sans',
+	'Overpass',
+	'Playfair Display',
+	'Poppins',
+	'Raleway',
+	'Red Hat Display',
+	'Roboto',
+	'Roboto Slab',
+	'Rubik',
+	'Source Sans Pro',
+	'Source Serif Pro',
+	'Space Mono',
+	'Texturina',
+	'Work Sans',
+);
+
+/**
+ * Register a curated selection of Google Fonts.
+ *
+ * @return void
+ */
+function blockbase_register_google_fonts() {
+	// Use jetpack's implementation of custom google fonts if it is already active
+	if ( method_exists( 'Jetpack', 'is_module_active' ) && Jetpack::is_module_active( 'google-fonts' ) ) {
```

Comment:
Would be a good idea to check when wpcomsh is auto-activating the Jetpack Google Fonts module, and move that to fire before this function, to make sure this check is working as expected. Otherwise this might run adding fonts, then the Jetpack module gets activated afterward, and we'd have duplicate fonts for a request.

Reply:
Good catch 👍 

Drafting a PR now

Reply:
@creativecoder 959-gh-Automattic/wpcomsh